src/modules/partition/core/PartitionLayout.cpp

```diff
@@ -128,6 +146,8 @@ PartitionLayout::addEntry( const QString& label, const QString& mountPoint, cons
     entry.partLabel = label;
     entry.partMountPoint = mountPoint;
     entry.partFileSystem = FileSystem::typeForName( fs );
```

Comment:
All the calls to FileSystem::typeForName() now use **localised** names; that means that sometimes you write "btrfs" and sometimes "Btrfs" depending on the language the user has selected. You can pass a localisation in, basically QStringList( {"C"} ). See findFS() in PartitionViewStep.cpp .. this also applies to other calls to typeForName(). (The function findFS() does some work to make sure it ends up with a canonical name -- maybe we should also just put the enum value into globalstorage, then you don't need to do any re-parsing of the name here)

Reply:
Oh OK, thanks for the tip ! I'm gonna export findFS into PartUtils then, and build on top of it.
Putting the enum value into globalstorage would be helpful for seeking the default filesystem, but I have to use findFS for every partition anyway, so, once more shouldn't harm (plus it would a one-shot use case, not necessarily worth it).